packages/core-transaction-pool/src/worker-script-handler.ts

```diff
@@ -18,12 +18,12 @@ export class WorkerScriptHandler implements Contracts.TransactionPool.WorkerScri
     }
 
     public async getTransactionFromData(
-        transactionData: Interfaces.ITransactionData | Buffer,
+        transactionDataDTO: Interfaces.ITransactionData | string,
```

Comment:
Preserve the old variable name. Previously it vaguely resembled a DTO but  it no longer does.